pkg/cache/cache_test.go

```diff
@@ -1489,12 +1456,12 @@ func TestCacheQueueOperations(t *testing.T) {
 					t.Fatalf("Running op %d: %v", i, err)
 				}
 			}
-			qCounts := make(map[string]map[string]int)
-			for _, cq := range cache.clusterQueues {
-				qCounts[cq.Name] = cq.admittedWorkloadsPerQueue
-			}
-			if diff := cmp.Diff(tc.wantQueueCounts, qCounts); diff != "" {
-				t.Errorf("Wrong active workloads counters for queues (-want,+got):\n%s", diff)
+			for _, lq := range queues {
+				queueAdmitted := cache.AdmittedWorkloadsLocalQueue(lq)
+				key := fmt.Sprintf("%s/%s", lq.Namespace, lq.Name)
+				if diff := cmp.Diff(tc.wantQueueCounts[key], queueAdmitted); diff != "" {
+					t.Errorf("Wrong active workload counters for queues from AdmittedWorkloadsLocalQueue:\n%s", diff)
+				}
```

Comment:
You can either do a single comparison after the loop ends, or you can add the key somewhere in the Error message. Otherwise we can't tell which queue counters didn't match